site-src/concepts/versioning.md

```diff
@@ -92,6 +92,45 @@ change across API versions.
 ### Major version (e.g. v0.x to v1.0)
 * There are no API compatibility guarantees when the major version changes.
 
+## Graduation Criteria
+
+### Resources
+
+#### Alpha -> Beta
+A resource to graduate from alpha to beta must meet the following criteria:
+
+* Implemented by several implementations
+* Initial conformance tests are in place
```

Comment:
IMHO `Initial` may be misunderstood by implementors. What conformance tests are considered `initial`?

Reply:
Updated, thanks!